_docs/reference/release-notes.md

```diff
@@ -8,4 +8,17 @@ layout: docs
 type: markdown
 ---
 
-TBD
+## Istio v0.1
+
+Istio v0.1 is the initial release of Istio. It works in a single Kubernetes cluster with a small number of services. Istio v0.1 supports the following features:
+- Installation of Istio into a Kubernetes namespace with a single command.
+- Injection of Envoy proxies into Kubernetes pods using `istioctl kube-inject`.
+- Automatic traffic capture for Kubernetes pods using iptables.
+- In-cluster load balancing for HTTP, gRPC, and TCP traffic.
+- Basic Kubernetes Ingress and Egress support.
+- Fine-grained traffic routing controls.
```

Comment:
Qualify again. A/B testing, Canary releases, red/black deployments for services exposed by ingress as well as internal services